apps/aecore/src/aec_db.erl

```diff
@@ -146,6 +155,12 @@ tables(Mode) ->
    , ?TAB(aec_discovered_pof)
     ].
 
+tables_created_if_not_present(Mode) when Mode==ram; Mode==disc ->
+    tables_created_if_not_present(expand_mode(Mode));
+tables_created_if_not_present(Mode) ->
+    [?TAB(aec_tx_gc)
```

Comment:
Do I understand correctly this is meant to move to `tables_manadatory` for 3.0? Could you please ensure there is ticket in Pivotal Tracker tracking this?

Reply:
This is not critical, and creation of this table is fine if it is not present. But, for clarity, we may move this to `tables_mandatory` for 3.0. This will block upgrade from `2.3.0` to `3.0.0` though?!

Reply:
> This will block upgrade from `2.3.0` to `3.0.0` though?!

Yes.

Reply:
I have no strong opinion here. I would keep it as is, until we find it conflicting with some introduced changes. But I can create a ticket if you find that appropriate @lucafavatella .

Reply:
I am just concerned of keeping complex code path. Ok, no need to.